test-tools/wamr-ide/VSCode-Extension/README.md

```diff
@@ -0,0 +1,19 @@
+# Intro
+
+### An integrated development environment for WASM.
+
+# How to debug this extension
```

Comment:
Do you mean "How to debug wasm application with this extension"? Here it seems that we want to debug the extension itself

Reply:
It does to debug the extension, not the wasm application.